leshan-client-demo/src/main/java/org/eclipse/leshan/client/demo/LeshanClientDemo.java

```diff
@@ -373,8 +373,10 @@ public static void createAndStartClient(String endpoint, String localAddress, in
                         clientPrivateKey.getEncoded(), serverCertificate.getEncoded()));
                 initializer.setInstancesForObject(SERVER, new Server(123, 30, BindingMode.U, false));
             } else if (useOSCore) {
-                initializer.setInstancesForObject(SECURITY, oscoreOnly(serverURI, 123));
-                initializer.setInstancesForObject(OSCORE, new Oscore("11223344", "AA", "BB")); //Hardcoded values
+            	Oscore oscoreObject = new Oscore("11223344", "AA", "BB"); //Hardcoded values
+            	oscoreObject.setId(12345);
```

Comment:
if you want you can create an Oscore constructor which take an instanceID in parameter

Reply:
Yes, I will do that instead.

Reply:
Now done in commit ae73ab25b69513f1b63b8bf4e2c9fbc4ca931a50